pub_topic: "/msis/pointcloud/python"

stonefish: 
  enabled: true
  range_min: 0.5
  range_max: 50.0
  number_of_bins: 100
  # frame: "wamv/msis"
  frame: "alpha_rise/ping360_link"
  sub_topic: "/alpha_rise/msis/stonefish/data/image"

ping360:
  sub_topic: "/alpha_rise/echo"
  frame: "alpha_rise/ping360_link"